.github/workflows/test.yml

```diff
@@ -6,75 +6,44 @@ on:
       - "!dependabot/**"
   pull_request:
   workflow_dispatch:
+
 jobs:
-  jest:
-    name: Jest (Node v${{ matrix.node }})
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        node: [14, 16, 17]
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Load Node version ${{ matrix.node }}
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ matrix.node }}
-          cache: npm
-      - name: Npm Install
-        run: npm ci
-      - name: Run Jest
-        run: npm run test:ci
-      - name: Run Coveralls
-        uses: coverallsapp/github-action@1.1.3
-        if: matrix.node == '16'
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          path-to-lcov: ./reports/lcov.info
+  tests:
+    uses: ljharb/actions/.github/workflows/node.yml@main
+    with:
+      range: '>= 0.4'
+      type: majors
+      command: npm run tests-only
+
   flow:
     name: Flow type checking
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Load Node
-        uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: ljharb/actions/node/install@main
```

Comment:
what does this do that's different from the standard `actions/setup-node`?

Reply:
It supports a wider range of node versions, and handles a lot of edge cases for npm installing in older ones. It also uses nvm to install node, and `nvm install-latest-npm`, which is the only way I'm aware of to reliably get the latest possible working version of npm on older nodes.

Reply:
Easy way to sneak a backdoor in lol